adafruit_ducky.py

```diff
@@ -128,76 +127,76 @@ def __init__(
         self.layout = layout
         self.lines = []
         self.default_delay = 0
-        self.last = 0
+        self.prev_line = None
+        self.next_index = 0
+        self.wait_until = 0
+        self.repeat = 0
 
         with open(filename, "r") as duckyscript:
             for line in duckyscript:
-                self.lines.append(line[:-1].rstrip("\r"))
+                line = line.strip(" \n\r")
```

Comment:
Is there a specific purpose for the leading space in this strip string `" \n\r"`?

I think that will cut out spaces at the end of the line, even if the user has intentionally put one there. 

If possible I think it'd be best to not cut out any characters they have put in the code. If they decide they didn't want the space it should get removed from their code, not have the library modified to ignore it.

I think this line would do what we want if written like this without the leading space:
```
line = line.strip("\n\r")
```
I'll try to test this again later on tonight.

Reply:
Possibly, but having significant whitespace at the end of lines seems not to be the best thing (anywhere, not just in ducky scripts :smile:)... that's what SPACE is for :smile: 

Either way, not having a space there could result in empty commands, which then need to be handled correctly, especially if the space is at the beginning of a line.

Reply:
The latest diff should account for this.